lib/components/ButtonLink/ButtonLink.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ReactNode } from 'react';
+import {
+  ButtonRenderer,
+  ButtonRendererProps,
+} from '../ButtonRenderer/ButtonRenderer';
+import {
+  useLinkComponent,
+  LinkComponentProps,
+} from '../BraidProvider/BraidProvider';
+
+export interface ButtonLinkProps
+  extends Omit<ButtonRendererProps, 'children'>,
+    Omit<LinkComponentProps, 'className' | 'style'> {
+  children?: ReactNode;
+}
+
+export const ButtonLink = ({
+  children,
+  weight = 'regular',
+  loading = false,
```

Comment:
These defaults should be owned by the `ButtonRenderer`

Reply:
This was copied over from `Button`, so I'll fix it there too.

Reply:
Fixed.